src/vs/workbench/browser/quickopen.ts

```diff
@@ -247,20 +248,38 @@ export class EditorQuickOpenEntry extends QuickOpenEntry implements IEditorQuick
 	}
 
 	public run(mode: Mode, context: IEntryRunContext): boolean {
-		if (mode === Mode.OPEN) {
+		const hideWidget = mode === Mode.OPEN;
+
+		let backgroundOpts;
```

Comment:
I suggest to move this into the other if check where we actually care about backgroundOpts. It is not being used in the scope otherwise.